pkg/scheduler/scheduler_test.go

```diff
@@ -848,10 +849,10 @@ func Test_UnionedGVKs(t *testing.T) {
 			want: map[framework.GVK]framework.ActionType{},
 		},
 		{
-			name:    "plugins with default profile",
+			name:    "plugins with default profile (InPlacePodVerticalScaling: disabled)",
 			plugins: schedulerapi.PluginSet{Enabled: defaults.PluginsV1.MultiPoint.Enabled},
 			want: map[framework.GVK]framework.ActionType{
-				framework.Pod:                   framework.All,
+				framework.Pod:                   framework.Add | framework.UpdatePodLabel | framework.Delete,
```

Comment:
Oh, this could potentially be very significant, as Pods have lots of status updates.
We wouldn't be able to observe this on the integration benchmarks (unless we add a case with status updates).

But it might increase the throughput in the load tests https://perf-dash.k8s.io/#/?jobname=gce-5000Nodes&metriccategoryname=Scheduler&metricname=LoadSchedulingThroughput&TestName=load

@macsko FYI, for when you run our internal scale tests too.

Reply:
Exactly, it's the core motivation that we reduce the number of events in-tree plugins subscribing and thus reduce the amount of events that the scheduling queue has to store as `inFlightEvents`.